lib/testing/src/lib/core/pages/form/widgets/widget.ts

```diff
@@ -112,4 +113,8 @@ export class Widget {
     peopleWidget(): PeopleWidget {
         return new PeopleWidget();
     }
+
+    tab(): Tab {
```

Comment:
Why a method and not just a member like: tab: Tab = new Tab() ?

Reply:
I know it's like that in many places, just asking why

Reply:
for sure tab() method could return the same instance every time, not creating a new one, but the entire class is like this, so i wouldn't change it only for tabs. Maybe the other methods from this class could be changed as well, didn't check them.